stacks/apps/augury.yml

```diff
@@ -739,6 +750,20 @@ Resources:
         - { Key: prx:ops:cloudwatch-log-group-name, Value: !Ref WorkerTaskLogGroup }
         - { Key: prx:dev:family, Value: Dovetail }
         - { Key: prx:dev:application, Value: Augury }
+  AugurySlowJobsProducedErrorsAlarm:
```

Comment:
Is there a functional approach here?  Something like a template include that defines all the alarms / filters and takes in an environment and log group ARN as parameters?

Reply:
Hmm... I can see that - but I'm also fine with a bit of code duplication in this case.  CFN templates are pretty verbose to begin with!